packages/modules/itwin-viewer-react/public/locales/en/iTwinViewer.json

```diff
@@ -2,5 +2,8 @@
   "components": {
     "tree": "Tree View",
     "properties": "Properties"
+  },
+  "iModels": {
+    "emptyIModel": "This iModel has no data. Please visit <a href={{bridgePortal}} rel=\"noopener noreferrer\" target=\"_blank\">the Synchronization Portal</a> to add data connections to your iModel and synchronize changes."
```

Comment:
unfortunately the imjs MessageBoxRenderer is sanitizing the html with the DOMPurify library. It is removing the target attr even if we have noopener and noreferrer set for rel. I've been in a "discussion" with the maintainer about why this is problematic, but I don't believe any changes will be made: https://github.com/cure53/DOMPurify/issues/317#issuecomment-761088912

I will have to ask imjs core to add a custom hook to allow this attr. In the meantime, links in their messageboxes will always open as self, including this one :-(